pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -2605,21 +2606,22 @@ var _ = Describe("v1beta1 inference service controller", func() {
 					},
 					WildcardPolicy: routev1.WildcardPolicyNone,
 				},
-				Status: routev1.RouteStatus{
-					Ingress: []routev1.RouteIngress{
-						{
-							Host: "raw-auth-default.example.com",
-							Conditions: []routev1.RouteIngressCondition{
-								{
-									Type:   routev1.RouteAdmitted,
-									Status: v1.ConditionTrue,
-								},
+			}
```

Comment:
is this for inferencegraph test?

Reply:
Nope. I updated the CRD test file `test/crds/route.openshift.io_routes.yaml`. This unit was failing after updating that CRD. I had to do these changes to make it pass again.

Something was wrong with the old CRD definition. The kube client typically doesn't save the `Status` field. So, it made sense to me that the unit started failing. To fix, I had to do these changes to fist create the resource with `k8sClient.Create()` and later update the status with `k8sClient.Status().Update()`.